src/wp-includes/blocks.php

```diff
@@ -40,6 +40,221 @@ function unregister_block_type( $name ) {
 	return WP_Block_Type_Registry::get_instance()->unregister( $name );
 }
 
+/**
+ * Removes the block asset's path prefix if provided.
+ *
+ * @since 5.5.0
+ *
+ * @param string $asset_handle_or_path Asset handle or prefixed path.
+ *
```

Comment:
There should be no empty line between @param and @return tags, see https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#docblock-formatting.

Reply:
Thanks for sharing the link, all DocBlock comments should be updated.